### ISSUE TITLE
no steel

### ISSUE THREAD

**Issue by jhilrd** [Melonslise/Locks#97]
Cannot make any steel item because there is no steel? Do I need a seperate mod?

**Comment by Eric-01** [comment]
Yes, you need, and it is not a bug. Any mod that registers its ingots under Forge tag `forge:ingots/steel` will work with Locks mod out of the box. Others can be customized with data packs.

**Comment by Greg-21** [comment]
Yes, you need a separate mod (and it is not a bug). Any mod that registers its ingots under Forge tag forge:ingots/steel will work with Locks mod out of the box. Others can be customized with data packs.

**Comment by Greg-21** [comment]
Yes, you need a separate mod (and it is not a bug). Any mod that registers its steel ingots under Forge tag `forge:ingots/steel` and steel nuggets under `forge:nuggets/steel` will work with Locks mod out of the box. Others can be customized with data packs.

**Comment by NixityNull** [comment]
there is stilll currently an issue where steel lockpick recipe crafts an iron lockpick though.
